src/libs/ReportUtils.js

```diff
@@ -682,12 +683,12 @@ function buildOptimisticIOUReport(ownerEmail, userEmail, total, chatReportID, cu
         cachedTotal: formattedTotal,
         chatReportID,
         currency,
-        hasOutstandingIOU: true,
+        hasOutstandingIOU: !isSendingMoney,
         managerEmail: userEmail,
         ownerEmail,
         reportID: generateReportID(),
         state: CONST.REPORT.STATE.SUBMITTED,
-        stateNum: 1,
+        stateNum: isSendingMoney ? 2 : 1,
```

Comment:
Also here, I am not sure what the stateNum signals here, we can find in the backend I am sure, but we should add some info here I assume.

Reply:
Bumping this one